fossas/fossa-cli
# [ANE-1639] add metadata to vendored dependencies

# Overview

This PR adds a metadata field for vendored dependencies. This will enable vendored dependencies to be specified with a homepage and description.

## Acceptance criteria

A homepage and description is supported for vendored dependencies

## Testing plan

- create a directory `bar` with a vendored dependency
- create a fossa-deps.yaml file like the following:
```
vendored-dependencies:
- name: metadata-test
  path: bar/
  version: 1.10.0
  metadata:
    homepage: https://fossa.com
    description: "metadata-test"
```
- run `cabal run fossa -- analyze /path/to/folder`
- navigate to the project created in FOSSA, and click on dependencies, if you click on the dependency you should see the homepage and description populated with your values

## Risks

n/a

## Metrics

n/a

## References

https://fossa.atlassian.net/browse/ANE-1639

## Checklist
